DSCResources/MSFT_xADRecycleBin/Tests/ResourceDesignerTests.ps1

```diff
@@ -0,0 +1,2 @@
+Test-xDscResource xADRecycleBin -Verbose
```

Comment:
This is great, we should just wrap it inside pester test.
It will be something like:

`ResourceDesigner.Tests.ps1` (note pester naming convention for tests is `*.Tests.ps1`)
```powershell
Describe 'xADRecycleBin' {
  Context 'xDscResouceDesigner' {
    Test-xDscResource xADRecycleBin -Verbose | Should Be $True
  }
}
```